DfE.FindInformationAcademiesTrusts.Data/Repositories/PipelineAcademy/IPipelineEstablishmentRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DfE.FindInformationAcademiesTrusts.Data.Repositories.PipelineAcademy
+{
+    public interface IPipelineEstablishmentRepository
+    {
+        Task<PipelineEstablishment[]?> GetPipelineFreeSchoolProjectsAsync(string trustReferenceNumber);
```

Comment:
Can we make these arrays not nullable? I'd always expect to see an empty array rather than null if there are no values.